stories/assets/img/private_tab_pagearea_ddgicon.svg

```diff
@@ -1,36 +1 @@
w.w3.org/2000/svg"><defs><filter x="-2.5%" y="-2.6%" width="105.1%" height="107.1%" filterUnits="objectBoundingBox" id="a"><feOffset dy="10" in="SourceAlpha" result="shadowOffsetOuter1"/><feGaussianBlur stdDeviation="10" in="shadowOffsetOuter1" result="shadowBlurOuter1"/><feColorMatrix values="0 0 0 0 0.0352941176 0 0 0 0 0.0352941176 0 0 0 0 0.0352941176 0 0 0 0.2 0" in="shadowBlurOuter1" result="shadowMatrixOuter1"/><feMerge><feMergeNode in="shadowMatrixOuter1"/><feMergeNode in="SourceGraphic"/></feMerge></filter></defs><g transform="translate(19.5 9.5)" filter="url(#a)" fill="none" fill-rule="evenodd"><path d="M75 37.5C75 58.213 58.213 75 37.5 75S0 58.213 0 37.5 16.787 0 37.5 0 75 16.787 75 37.5z" fill="#FF3D00" fill-rule="nonzero"/><path d="M41.25 22.873c-1.129-1.121-2.813-1.684-4.688-2.059-.746-.937-1.874-1.875-3.558-2.812C30 16.5 26.44 15.746 23.064 16.31h-.753c-.184 0-.373.19-.747.19.373 0 1.875.747 3.004 1.122-.564.38-1.502.38-2.066.754h-.191l-.375.373c-.184.373-.373.746-.373.938 2.439-.192 5.998 0 8.628.746-2.066.191-3.941.755-5.252 1.502-.937.564-1.875 1.128-2.439 2.066-2.248 2.44-3.186 6.562-2.44 11.059.938 5.06 4.504 21.378 6.38 30.564l.474 3.007s6.6.743 10.586.743c2.231 0 6.054.585 6.915-.353-.1.02-1.133-1.171-1.48-2.086a103.307 103.307 0 0 0-2.62-4.872c-2.249-4.687-4.688-11.066-3.56-15.19.184-.747.184-3.942.747-4.315 4.879-3.186 4.496-.184 6.562-1.502.938-.746 1.875-1.684 2.248-2.813 1.31-4.303-1.685-11.994-5.062-15.37z" fill="#FFF" fill-rule="nonzero"/><circle stroke="#FFF" stroke-width="2.7" cx="37.5" cy="37.5" r="31.875"/><path d="M28.125 32.066c-1.129 0-2.248.938-2.248 2.248 0 1.122.937 2.248 2.248 2.248 1.129 0 2.248-.937 2.248-2.248-.182-1.128-1.12-2.248-2.248-2.248zm.938 2.057c-.374 0-.565-.184-.565-.564 0-.373.191-.557.564-.557.374 0 .565.184.565.557 0 .382-.373.564-.564.564zm13.689-2.994c-1.129 0-1.875.937-1.875 1.875 0 1.121.937 1.875 1.875 1.875 1.121 0 1.875-.938 1.875-1.875 0-.938-.938-1.875-1.875-1.875zm.746 1.683c-.184 0-.564-.19-.564-.564 0-.184.191-.564.564-.564.191 0 .565.191.565.564 0 .373-.192.565-.565.565z" fill="#0277BD" fill-rule="nonzero"/><path d="M36.19 59.436s-8.065-4.315-8.254-2.622c-.184 1.684 0 8.811.937 9.375.938.565 7.691-3.566 7.691-3.566l-.375-3.187zm3.185-.182s5.434-4.13 6.754-3.941c1.121.19 1.494 8.81.373 9.183-1.129.38-7.316-2.248-7.316-2.248l.189-2.994z" fill="#8BC34A" fill-rule="nonzero"/><path d="M34.688 59.436c0 2.812-.374 3.94.746 4.314 1.129.191 3.566 0 4.314-.564.754-.565.191-4.123-.184-4.872-.189-.562-4.877.002-4.877 1.122" fill="#689F38" fill-rule="nonzero"/><path d="M34.314 42.752c.184-1.31 3.75-3.941 6.19-4.123 2.432-.191 3.185-.191 5.244-.565 2.066-.564 7.316-1.875 8.818-2.439 1.495-.746 7.684.373 3.37 2.813C56.06 39.566 51 41.44 47.25 42.56c-3.559 1.128-5.809-1.122-7.127.753-.937 1.495-.184 3.37 4.13 3.75 5.81.557 11.624-2.628 12.188-.937.557 1.684-5.068 3.75-8.629 3.934-3.376 0-10.503-2.248-11.44-2.997-.938-.753-2.249-2.437-2.058-4.312" fill="#FFCA28" fill-rule="nonzero"/></g></svg>
```

Comment:
I'm not sure what the formatting guidelines are for svgs. Deferring

Reply:
I literally copy-pasted what we have in browser-laptop. @petemill should we have this as a real icon in our icon set? this seems more like an image to me but I'm ok following up on this

Reply:
I don't want to complicate this but both these could possibly be brought in as a full color icon component. I worked them up in the same structure as other icons here: https://share.goabstract.com/3522057a-281c-48e4-bedc-33daf84cfd7c